controllers/webhooks/application_webhook_test.go

```diff
@@ -102,77 +86,4 @@ var _ = Describe("Application validation webhook", func() {
 		})
 	})
 
-	Context("Update Application CR fields", func() {
-		It("Should update non immutable fields successfully and err out on immutable fields", func() {
-			ctx := context.Background()
-
-			hasApp := &appstudiov1alpha1.Application{
-				TypeMeta: metav1.TypeMeta{
-					APIVersion: "appstudio.redhat.com/v1alpha1",
-					Kind:       "Application",
-				},
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      HASAppName,
-					Namespace: HASAppNamespace,
-				},
-				Spec: appstudiov1alpha1.ApplicationSpec{
-					DisplayName: DisplayName,
-					Description: Description,
-					AppModelRepository: appstudiov1alpha1.ApplicationGitRepository{
-						URL: "https://github.com/testorg/petclinic-app",
-					},
-					GitOpsRepository: appstudiov1alpha1.ApplicationGitRepository{
-						URL: "https://github.com/testorg/gitops-app",
-					},
-				},
-			}
-
-			Expect(k8sClient.Create(ctx, hasApp)).Should(Succeed())
-
-			// Look up the has app resource that was created.
-			// These tests do not go through reconcile, so no need to check reconcile logic here
-			hasAppLookupKey := types.NamespacedName{Name: HASAppName, Namespace: HASAppNamespace}
-			fetchedHasApp := &appstudiov1alpha1.Application{}
-			Eventually(func() bool {
-				k8sClient.Get(ctx, hasAppLookupKey, fetchedHasApp)
-				return !reflect.DeepEqual(fetchedHasApp, &appstudiov1alpha1.Application{})
-			}, timeout, interval).Should(BeTrue())
-
-			// Update the hasApp resource
-			fetchedHasApp.Spec.DisplayName = "newname"
-			fetchedHasApp.Spec.Description = "New Description"
-			fetchedHasApp.Spec.AppModelRepository.URL = "newurl"
-			err := k8sClient.Update(ctx, fetchedHasApp)
-			Expect(err).Should(HaveOccurred())
-			Expect(err.Error()).Should(ContainSubstring("app model repository cannot be updated"))
-
-			// revert appmodel but update gitops
-			fetchedHasApp.Spec.AppModelRepository.URL = "https://github.com/testorg/petclinic-app"
-			fetchedHasApp.Spec.GitOpsRepository.URL = "https://github.com/testorg/petclinic-app"
-
-			err = k8sClient.Update(ctx, fetchedHasApp)
-			Expect(err).Should(HaveOccurred())
-			Expect(err.Error()).Should(ContainSubstring("gitops repository cannot be updated"))
-
-			// Delete the specified resource
-			deleteHASAppCR(hasAppLookupKey)
-		})
-	})
-
 })
-
-// deleteHASAppCR deletes the specified hasApp resource and verifies it was properly deleted
-func deleteHASAppCR(hasAppLookupKey types.NamespacedName) {
```

Comment:
I think this is still being used in the other tests

Reply:
It wasn't used - it was duplicated with an identical function in the application controller tests